src/main/java/seedu/loyaltylift/logic/parser/ParserUtil.java

```diff
@@ -137,6 +152,19 @@ public static CustomerType parseCustomerType(String customerType) throws ParseEx
         }
         return type;
     }
+    /**
+     * Parses {@code String status} into a {@code Status}.
+     */
+    public static Status parseStatus(String statusString) throws ParseException {
+        requireNonNull(statusString);
+        Status status;
+        try {
+            status = Status.fromString(statusString);
+        } catch (IllegalArgumentException | NullPointerException e) {
```

Comment:
`NullPointerException` should not be necessary here due to the check on line 159